config/storage_services.yml

```diff
@@ -0,0 +1,28 @@
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
```

Comment:
Make this an ENV so we can use a shared directory -- workers need to access these files to copy them to figgy storage.